### PR TITLE
#160617650 List Bookmarked articles

### PR DESCRIPTION
**Pull request by kcharles52**
#### What does this PR do?
- Enables users to view a list of articles they have bookmarked
#### Description of Task to be completed?
- Enable authenticated users to view bookmarked articles
#### How should this be manually tested?
- When you click on profile picture on the navbar a drop-down menu appears. 
- Select bookmarks
#### What are the relevant pivotal tracker stories?
[#160617650]

#### Screenshoot
screen shot 2018-11-22 at 16 50 12

### CODE REVIEW

File: src/views/bookmark/bookmarkArticleView.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        </div>
+      </div>
+      <div className="removeLink">
+        <Button>Remove </Button>
+      </div>
+    </div>
+  ));
+
+  return (
+    <React.Fragment>
```
Comment (LehruAngela) [inline src/views/bookmark/bookmarkArticleView.js]: Change `React.Fragment` to just `Fragment` by destructuring it the way you would do for `Component`

File: src/views/bookmark/bookmarkArticleView.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          className="ArticleImage"
+        />
+        <div className="titleDiscription">
+          <h3>{bookmark.article_title}</h3>
+          <p>{bookmark.description} ...</p>
+        </div>
+        <div className="dateRead">
+          <div className="articleAuthor dtTab">{bookmark.author}</div>
+          <div className="dateTime dtTab">13th Nov 2018</div>
+          <div className="readTime dtTab">3 min read</div>
```
Comment (LehruAngela) [inline src/views/bookmark/bookmarkArticleView.js]: Why is the dateTime and readTime static? Let it be dynamic as you did for the articleAuthor

**Review by LehruAngela — CHANGES_REQUESTED** [review]

File: src/views/bookmark/bookmarkArticleView.js (hunk 1 above)
Comment (kcharles52) [reply src/views/bookmark/bookmarkArticleView.js]: I have changed it

**Comment by LehruAngela** [comment]
Add screenshots for this task here @kcharles52

**Comment by LehruAngela** [comment]
Rebase with develop @kcharles52

**Comment by LehruAngela** [comment]
Rebase with develop @kcharles52

**Pull request merged**